src/ui/index.js

```diff
@@ -1,24 +1,26 @@
 import Vue from 'vue'
-import vuetify from './plugins/vuetify'
-import capacitor from './plugins/capacitor'
+import {i18n} from '../lib/native/I18n'
 import App from './App'
-import { router } from './router'
-import store from './store'
+import capacitor from './plugins/capacitor'
 import i18nPlugin from './plugins/i18n'
-import {i18n} from '../lib/native/I18n'
+import vuetify from './plugins/vuetify'
+import {router} from './router'
+import store from './store'
 
 Vue.mixin(i18nPlugin)
 Vue.mixin(capacitor)
 
-i18n.setLocales(navigator.languages)
-i18n.load()
-
-const app = (global['Floccus'] = new Vue({
-  el: '#app',
-  store,
-  router,
-  vuetify,
-  render: (h) => h(App),
-}))
+const app = () => {
+  i18n.setLocales(navigator.languages)
+  i18n.load().then(() => {
+    window['floccus'] = global['Floccus'] = new Vue({
```

Comment:
It seems that `window['floccus']` and `global['Floccus']` are not useful, maybe they can be omitted.

Reply:
Ah, please leave those in. They're useful for debugging